### PR TITLE
dcerpc: use null for default ports

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/4475

### CODE REVIEW

**Comment by suricata-qa** [comment]
| field | test | baseline | % diff |
|--|--|--|--|
| | tlpr1_stats_chk |
| .detect.alert | 3109382 | 3962505 | 78.0% |

**Comment by suricata-qa** [comment]
| field | test | baseline | % diff |
|--|--|--|--|
| | tlpr1_stats_chk |
| .tcp.insert_list_fail | 219 | 255 | 86.0% |

File: rust/src/dcerpc/dcerpc.rs [hunk 1]
```diff
@@ -1378,10 +1378,9 @@ pub const PARSER_NAME: &'static [u8] = b"dcerpc\0";
 
 #[no_mangle]
 pub unsafe extern "C" fn rs_dcerpc_register_parser() {
-    let default_port = CString::new("[0:65355]").unwrap();
     let parser = RustParser {
         name: PARSER_NAME.as_ptr() as *const std::os::raw::c_char,
-        default_port: default_port.as_ptr(),
+        default_port: b"\0".as_ptr() as *const std::os::raw::c_char,
```
Comment (jasonish) [inline rust/src/dcerpc/dcerpc.rs:1383]: This is creating an empty string, not null.  `std::ptr::null()` can be used here for a real null value.

**Review by jasonish — CHANGES_REQUESTED** [review]
While an empty string might be good enough to get the same result, we should probably use a real null.

File: rust/src/dcerpc/dcerpc.rs (hunk 1 above)
Comment (inashivb) [reply rust/src/dcerpc/dcerpc.rs:1383]: Thanks. I thought this was null...

**Comment by inashivb** [comment]
Replaced with #6234

**Pull request closed without merging**